### PR TITLE
Find common free time

### LINKED ISSUE #61: Find a common time to be able to conduct a training session
-Filter by common time - high

### PR DESCRIPTION
**Pull request by faqihisaninja**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CommonTimeCommand.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+import java.time.LocalTime;
+import java.util.List;
+
+import seedu.address.model.Model;
+import seedu.address.model.student.CommonTimeFinder;
+import seedu.address.model.student.PredicateList;
+
+/**
+ * Finds the latest dismissal time of all the students stated in he keywords.
```
Comment (yejiadong) [inline src/main/java/seedu/address/logic/commands/CommonTimeCommand.java:13]: minor spelling of "the"

**Review by kerkpy — APPROVED** [review]
LGTM

**Review by yejiadong — APPROVED** [review]
LGTM.

File: src/main/java/seedu/address/logic/commands/CommonTimeCommand.java (hunk 1 above)
Comment (faqihisaninja) [reply src/main/java/seedu/address/logic/commands/CommonTimeCommand.java:13]: okay! fixed it.

**Pull request merged**